sklearn/tests/test_cross_validation.py

```diff
@@ -651,6 +657,7 @@ def test_cross_val_score():
     assert_raises(ValueError, cval.cross_val_score, clf, X_3d, y)
 
 
+@ignore_warnings
```

Comment:
There was some pandas warning I think. Let me check and revert if it doesn't catch anything.